odl/tomo/geometry/spect_geometry.py

```diff
@@ -0,0 +1,156 @@
+# Copyright 2014 - 2016 The ODL development group
+#
+# This file is part of ODL.
+#
+# ODL is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# ODL is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with ODL.  If not, see <http://www.gnu.org/licenses/>.
+
+"""SPECT geometry."""
+
+# Imports for common Python 2/3 codebase
+from __future__ import print_function, division, absolute_import
+from future import standard_library
+standard_library.install_aliases()
+from builtins import super
+
+# External
+import numpy as np
+
+# Internal
+import odl
+from odl.tomo.geometry.detector import Flat2dDetector
+from odl.tomo.geometry.geometry import (Geometry,
+                                        AxisOrientedGeometry)
+from odl.tomo.util.utility import perpendicular_vector
+
+__all__ = ('ParallelHole')
```

Comment:
This creates an all that looks like:

```
__all__ = ('P', 'a', 'r', 'a', 'l', 'l', 'e', 'l', 'H', 'o', 'l', 'e')
```

you need an extra `,` at the end to make this proper:

```
__all__ = ('ParallelHole',)
```